torchao/quantization/autoquant.py

```diff
@@ -227,9 +232,13 @@ def do_autoquant_bench(op, *args, **kwargs):
         graph = torch.cuda.CUDAGraph()
         with torch.cuda.graph(graph, stream=stream):
             op(*args, **kwargs)
-        if TORCH_VERSION_AFTER_2_3:
+        if TORCH_VERSION_AFTER_2_3 and not TORCH_VERSION_AFTER_2_5:
             from torch._inductor.runtime.runtime_utils import do_bench_gpu
             res = do_bench_gpu(lambda: graph.replay(), warmup=warmup, rep=rep, return_mode="median")
+        elif TORCH_VERSION_AFTER_2_5 and torch.cuda.is_available():
+            from torch._inductor.runtime.benchmarking import benchmarker
+            do_bench_gpu = benchmarker.benchmark_gpu
+            res = do_bench_gpu(lambda: graph.replay(), warmup=warmup, rep=rep, return_mode="median")
```

Comment:
The newest (i.e. `from torch._inductor.runtime.benchmarking import benchmarker`) block needs to go first, as from my understanding `TORCH_VERSION_AFTER_X_Y=True` always in fbcode.